spec.html

```diff
@@ -48948,102 +48918,105 @@ <h1>
   <emu-clause id="sec-relations-of-candidate-executions">
     <h1>Relations of Candidate Executions</h1>
 
-    <emu-clause id="sec-agent-order" aoid="agent-order">
-      <h1>agent-order</h1>
-      <p>For a candidate execution _execution_, _execution_.[[AgentOrder]] is a Relation on events that satisfies the following.</p>
+    <p>The following relations and mathematical functions are parameterized over a particular candidate execution and orders its events.</p>
+
+    <emu-clause id="sec-agent-order">
+      <h1>is-agent-order-before</h1>
+      <p>For a candidate execution _execution_, its <dfn>is-agent-order-before</dfn> Relation is the least Relation on events that satisfies the following.</p>
       <ul>
-        <li>For each pair (_E_, _D_) in EventSet(_execution_), _execution_.[[AgentOrder]] contains (_E_, _D_) if there is some Agent Events Record _aer_ in _execution_.[[EventsRecords]] such that _E_ and _D_ are in _aer_.[[EventList]] and _E_ is before _D_ in List order of _aer_.[[EventList]].</li>
+        <li>For each pair (_E_, _D_) in EventSet(_execution_), _E_ is-agent-order-before _D_ in _execution_ if there is some Agent Events Record _aer_ in _execution_.[[EventsRecords]] such that _E_ and _D_ are in _aer_.[[EventList]] and _E_ is before _D_ in List order of _aer_.[[EventList]].</li>
       </ul>
 
       <emu-note>
         <p>Each agent introduces events in a per-agent strict total order during the evaluation. This is the union of those strict total orders.</p>
       </emu-note>
     </emu-clause>
 
-    <emu-clause id="sec-reads-bytes-from" aoid="reads-bytes-from">
+    <emu-clause id="sec-reads-bytes-from">
       <h1>reads-bytes-from</h1>
-      <p>For a candidate execution _execution_, _execution_.[[ReadsBytesFrom]] is a mathematical function mapping events in SharedDataBlockEventSet(_execution_) to Lists of events in SharedDataBlockEventSet(_execution_) that satisfies the following conditions.</p>
+      <p>For a candidate execution _execution_, its <dfn>reads-bytes-from</dfn> function is a mathematical function mapping events in SharedDataBlockEventSet(_execution_) to Lists of events in SharedDataBlockEventSet(_execution_) that satisfies the following conditions.</p>
```

Comment:
While this makes more sense as a definition, practically speaking ecmarkup will be happier if you replace the `dfn` with an `em` and put `aoid="reads-bytes-from"` on the clause, so that it can tell that this is invoked. Ecmarkup expects things of the form `identifier()` to be associated with an AO.

Reply:
Done.